download-script.py

```diff
@@ -13,14 +14,33 @@
     print("no user in that name found\nRun program again")
     exit(1)
 
+seperator = ""
+if (platform.system() == "Linux"):
+    seperator = "/"
+
+elif(platform.system() == "Windows"):
```

Comment:
Seperators, this is one of the reasons the file systems are different and required different path formats.